torchci/pages/metrics.tsx

```diff
@@ -33,59 +33,6 @@ import TablePanel from "components/metrics/panels/TablePanel";
 import TimeSeriesPanel from "components/metrics/panels/TimeSeriesPanel";
 import { durationDisplay } from "components/TimeUtils";
 
-function MasterJobsRedPanel({ params }: { params: RocksetParam[] }) {
-  const url = `/api/query/metrics/master_jobs_red?parameters=${encodeURIComponent(
```

Comment:
Is this the only user of `master_jobs_red`? If this is the case, please delete respective sql query from rockset

Reply:
Good point. I'll double check and do so